src/app/logic/localStorage.js

```diff
@@ -0,0 +1,23 @@
+function addResult(rankings, gameInfo, amountOfResults) {
+  const { name, result, mode } = gameInfo;
+  // push gameinfo (only name and result, we don't need mode in this place) into the right mode in
+  // rankings
+  rankings[mode].push({ name, result });
+  // sort in descending order by points
+  rankings[mode].sort((a, b) => (a.result < b.result ? 1 : -1));
+  // remove last element if there's to many gameinfos in our game ranking
+  if (rankings[mode].length > amountOfResults) rankings[mode].pop();
```

Comment:
Something to do as a side note - create constants folder in which you would keep all the strings and numbers that you will be using but not modifying, it prevents from making mistakes and creating so called "magic numbers", that are numbers nobody knows why appeared in the code (why 3? Why not 5?).